source/process/sg_rest_markup.rst

```diff
@@ -173,23 +177,21 @@ Later on in the document, perhaps at the end of the paragraph that contains the
 
 .. _literal-blocks:
 
-Literal blocks
+Literal Blocks
 --------------
 
-In reStructuredText markup, the double colon marks the start of a section of literal text that corresponds to the HTML <pre> tag. However, the Sphinx processor applies syntax highlighting for Python to literal blocks, which is not always desired in Mattermost documentation.
-
-To use a literal block as originally intended in the reStructuredText specification, you must cheat a little, and use the Sphinx code-block directive with the language set to `none`. For example:
+To use a literal block with no syntax highlighting, use the Sphinx code-block directive with the language set to `none`. For example:
```

Comment:
Formatting on `none`, does this follow the guidelines / should we update guidelines if it's not covered?